ignite/cmd/plugin.go

```diff
@@ -18,33 +18,46 @@ import (
 	"github.com/ignite/cli/ignite/services/plugin"
 )
 
-// plugins hold the list of plugin declared in the config.
-// A global variable is used so the list is accessible to the plugin commands.
-var plugins []*plugin.Plugin
-
 const (
 	igniteCmdPrefix = "ignite "
 )
 
+// plugins hold the list of plugin declared in the config.
+// A global variable is used so the list is accessible to the plugin commands.
+var plugins []*plugin.Plugin
+
 // LoadPlugins tries to load all the plugins found in configuration.
 // If no configuration found, it returns w/o error.
 func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
-	cfg, _, err := parseLocalPlugins(rootCmd)
-	if err != nil {
-		// if binary is run where there is no plugins.yml, don't load
-		return nil
+	localCfg, _, err := parseLocalConfig(rootCmd)
+	// if binary is run where there is no cfg, don't load
+	if err == nil {
+		plugins, err = plugin.Load(ctx, localCfg)
+		if err != nil {
+			return err
+		}
 	}
 
-	// TODO: parse global config
+	globalCfg, _, err := parseGlobalConfig()
+	// if binary is run where there is no cfg, don't load
+	if err == nil {
+		globalPlugins, err := plugin.Load(ctx, globalCfg)
+		if err != nil {
+			return err
+		}
+		plugins = append(plugins, globalPlugins...)
+	}
```

Comment:
We should probably change `plugin.Load` signature so it takes a slice of `pluginsconfig.Plugin` instead of a `pluginsconfig.Config`. Thus we'll be able to call `plugin.Load` a single time, with a sorted list of plugins (with duplicates removed for instance).